package.json

```diff
@@ -17,22 +20,9 @@
   "devDependencies": {
     "eslint": "^7.7.0"
```

Comment:
Q: can't we bump eslint version by the occasion?

Reply:
updated :)